src/api/controllers/reportedCommentAdminController.js

```diff
@@ -0,0 +1,107 @@
+import models from '../models';
+import status from '../../helpers/constants/status.codes';
+import paginateUser from '../../helpers/generate.pagination.details';
+
+const generatePagination = paginateUser;
+
+const {
+  Comment, ReportedComment, User, Reason
+} = models;
+
+/**
+ * contains all method for an admin to read/delete/Block/unblock a comment
+ *
```

Comment:
are you deleting, blocking or unblocking?

Reply:
this is the controller where all the methods for doing those actions will live